src/seb/cli/run.py

```diff
@@ -114,3 +116,10 @@ def run_benchmark_cli(
         raise_errors=not ignore_errors,
     )
     benchmark_result.to_disk(output_path)
+    benchmark_result.meta.name = f"NEW: {benchmark_result.meta.name}"
+    full_results = benchmark.evaluate_models(
+        models=get_all_models(), use_cache=True, run_model=False, cache_dir=None
+    )
```

Comment:
Hmm it is problematic if this fails. I expect the error would be quite unintuitive.

You run model A, but then get an error on Model B. One option might be to add "raise_errors = False".

Reply:
I can't set it, it says I can't use raise_errors=False with use_cache=True

Reply:
Yea. That was my fail safe. I think it can be converted into a warning.